charmcraft.yaml

```
@@ -0,0 +1,24 @@
+name: kubernetes-e2e
+type: charm
+title: 
+summary: Run e2e validation of a cluster's conformance
+description: |
+  Deploy the Kubernetes e2e framework and validate the conformance of a deployed
+  Kubernetes cluster.
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
```

I think we need to widen this to run on focal and jammy on amd64 and arm64

```suggestion
  - build-on:
    - name: ubuntu
      channel: "20.04"
      architectures: [amd64]
    run-on:
    - name: ubuntu
      channel: "20.04"
      architectures: [amd64, arm64]
    - name: ubuntu
      channel: "22.04"
      architectures: [amd64, arm64]
```